src/utils/buildFetchOptions.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import ProxyAgent from 'proxy-agent'
+
+const defaultProxy = process.env.https_proxy || process.env.http_proxy || ''
+
+export const buildAgent = (proxy: string = defaultProxy) =>
+  proxy ? new ProxyAgent(proxy) : ''
+
+export const buildFetchOptions = (options: { proxy?: string } = {}) => {
```

Comment:
Add a type for the return 
```suggestion
export const buildFetchOptions = (options: { proxy?: string } = {}): { agent: Object } => {
```

Reply:
Hi @carloscuesta,
I'll get on those changes this weekend. 
😀

Reply:
Awesome thanks ❤️